onto_tool/onto_tool.py

```diff
@@ -99,6 +99,10 @@ def configure_arg_parser():
                                'any properties other than rdf:type will be annotated. '
                                'Will override any existing rdfs:isDefinedBy annotations '
```

Comment:
I believe this line is no longer true if --retain-definedBy is used.

Reply:
Corrected, and documentation adjusted.